extensions/tika/runtime/src/main/java/io/quarkus/tika/runtime/TikaConfiguration.java

```diff
@@ -12,11 +12,36 @@
 @ConfigRoot(phase = ConfigPhase.BUILD_AND_RUN_TIME_FIXED)
 public class TikaConfiguration {
     /**
-     * The path to the tika-config.xml
+     * Path to the tika-config.xml
      */
     @ConfigItem
     public Optional<String> tikaConfigPath;
```

Comment:
I know this isn't actually part of the PR, but it's better for paths to be of type `Path`; this adds a bit of extra validation.

Reply:
@dmlloyd For the moment it is expected to be on the class path only and the extension does validates this value and throws the exception if the resource does not exist. No file resources are supported. So having `Path` would move this validation to the custom converter (which would to override the implicit one shipped with Quarkus :-) ). For the moment I'm keeping it as String - but would have no problems if you will still prefer `Path`.

Reply:
Ah okay.  Using `String` is fine then, but you'd better clarify that in the doc, something like this:

```java
The resource path within the application artifact to the {@code tika-config.xml} file.
```

Reply:
@dmlloyd done, I'd never be able to come up myself with such a compact yet complete doc sentence :-)